weave-js/src/errors.ts

```diff
@@ -1 +1,23 @@
+import {ApolloError} from '@apollo/client';
+import {get} from 'lodash';
+
 export class UseNodeValueServerExecutionError extends Error {}
+
+export function extractErrorMessageFromApolloError(
```

Comment:
these utils are copied from core. I don't actually use this "extract err msg" one in this PR but figured it'll prob be useful in future